app/views/admin/conferences/_todo_list.html.haml

```diff
@@ -23,7 +23,7 @@
       Set up call for papers
   %li{'class'=>"list-group-item #{class_for_todo(conference_progress['venue'])}"}
     %span{'class'=>icon_for_todo(conference_progress['venue'])}
-    - if can? :update, @conference.venue
+    - if can? :update, @conference
```

Comment:
Technically this should be checking if the user can create or update a venue. 
If we do that (due to the create check where we will initialize a venue object associated with conference), we would need to reload the conference, so that it shows the right link. While we have a venue object, the link will redirect to edit, even though that object is not saved, and does not contain the data we need.